include/crow/json.h

```diff
@@ -1954,13 +1986,20 @@ namespace crow
             }
 
         public:
-            std::string dump() const
+            std::string dump(const int indent, const char separator = ' ') const
```

Comment:
Maybe set a default value also for `indent`?

Reply:
setting a default value for `indent` conflicts with the virtual function `dump` of type `returnable`, so I used to keep the previous interface.